docs/getting-started.md

```diff
@@ -4,26 +4,192 @@
 
 ### Building from source
 
-Prerequisites: The [Go](https://golang.org/doc/install) compiler **version 1.15** (we use
-go modules) or later and GNU make. The go bin path (usually `$HOME/go/bin`) should be
-in your system `$PATH`.
+#### Prerequisites
 
-Run `make install` to build the code. The `regen` binary serves as both the blockchain daemon, as well as
+**Minimum Hardware Requirements**
+
+- 8GB RAM
+- 4vCPUs
+- 200GB Disk space
+
+**Software Requirements** 
+
+Before beginning make sure you have the [Go](https://golang.org/doc/install) compiler **version 1.15** (we use
+go modules) or later and GNU make installed and add the go bin path (usually `$HOME/go/bin`) should be in your system `$PATH`. To learn more about adding commands to your path please visit [linuxize.com](https://linuxize.com/post/how-to-add-directory-to-path-in-linux/). We recommend using Ubuntu 18.04 or higher, the official images can be found at [Ubuntu.com](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview)
+
+---
+
+#### Download and Install Regen Ledger
+
+Install git and build-essentials for ubuntu
+```sh
+sudo apt update
+```
+```sh
+sudo apt install git build-essential jq -y
+```
+
+Download go and extract tarball
+```sh
+wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
+```
+```sh
+tar -xvf go1.15.2.linux-amd64.tar.gz
+```
+```sh
+sudo mv go /usr/local
+```
+
+Add go to local path
+```sh
+echo "" >> ~/.profile
+```
+```sh
+echo 'export GOPATH=$HOME/go' >> ~/.profile
+```
+```sh
+echo 'export GOROOT=/usr/local/go' >> ~/.profile
+```
+```sh
+echo 'export GOBIN=$GOPATH/bin' >> ~/.profile
+```
+```sh
+echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> ~/.profile
+```
+
+```sh
+. ~/.profile
+```
+
+Check to make sure you have the correct version of installed **version 1.15** or higher
+```sh
+go version
+```
+
+Next, we download the official Regen ledger github repository
+```sh
+cd $HOME
+```
+```sh
+git clone https://github.com/regen-network/regen-ledger $GOPATH/src/github.com/regen-network/regen-ledger
+```
+```sh
+cd $GOPATH/src/github.com/regen-network/regen-ledger
+```
+
+Run `make install` to build the code.
+```sh
+make install
+```
+
+Once installed the `regen` binary serves as both the blockchain daemon, as well as
 the command line blockchain client.
 
+## Creating Keys
+
+Run this command replacing `[moniker]` with the name of your key(you get to choose this). 
+
+::: **IMPORTANT!**
```

Comment:
This doesn't render the container properly. It should be something like: 
```suggestion
::: tip IMPORTANT
```

The first word is how vuepress selects the color of the container. Options are "tip", "warning", "danger". You can see previews of them all [here](https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers).

Reply:
btw- if you want to preview this all locally, you can cd into the docs directory and run `yarn install && yarn run dev`.